### PR TITLE
Representative Update

### PR DESCRIPTION
**Pull request by TheBronJameOffical**
## About The Pull Request
Updates all rep related stuff. You WILL be eatin good tonight you WILL play the representative. 

- K Corp
  - Added 3 New ERTs.
    - Class 1: Consists of Class 1employees, can either have the Kbatong or Kaxe along with a shield. Same stats as Warp L1 Employees.
    - Class 3: Excision Staff, 80s and comes with the KSpear weapon. Also has a shield.
    - Asset Protection Team: The "leave our loot alone or die" team. Consists of super powered up Class 3 K-corp goons and K-Corp drones.
  - Replaced Crit Up syringe with Health Boost.
- R Corp
  - Added 4 Researches to purchase new weapons from R-Corp rep.
    - Red Weapon: Unlocks (almost) all R-Corp guns with only RED damage. (600 Ahn)
    - Single Phase: Unlocks Heavy Red gun as well as WHITE, BLACK, and PALE guns. (700 Ahn)
    - Multiphase: Unlocks R-Corp Lawnmower and Minigun with R,W, and B bullets as well as High Freq Blade. (800 Ahn)
    - Experimental: Unlocks Rush Dagger and Reindeer Staff. (900 Ahn)
  - Removed Rush Dagger from base Console
  - Lowered price of Lawnmower 2100 to 500 Ahn

Side Shenanigans:
- Remade Reject Message from spawners to be flexible and not default on R-corps message.

## Why It's Good For The Game
Lets see.

## Changelog
:cl:
add: Added Rep related things
balance: R-Corp rep is a bit stronger
soundadd: added KSpear sounds
/:cl:

### CODE REVIEW

File: ModularTegustation/tegu_items/representative/research/kcorp.dm [hunk 1]
```diff
@@ -61,13 +61,28 @@
...
+//Health booster
+/datum/data/lc13research/healthboost
+	research_name = "Repeatable: Health Booster Syringe "
+	research_desc = "Hey man, its Joe from research. <br>We've got a new item over here, something about increasing your overall health? <br>Management needs them tested, and what better place than here? You can buy more anytime."
 	cost = AVERAGE_RESEARCH_PRICE
 	corp = K_CORP_REP
 
-/datum/data/lc13research/critincrease/ResearchEffect(obj/structure/representative_console/caller)
-	new /obj/item/kcrit(get_turf(caller))
+/datum/data/lc13research/healthboost/ResearchEffect(obj/structure/representative_console/caller)
```
Comment (Kitsunemitsu) [inline ModularTegustation/tegu_items/representative/research/kcorp.dm:71]: After reading what it does, I realize I made a massive design mistake.
Making this repurchasable will lead to bullshit like having 700 HP.
Can you add a ..() in after this line to make it not repurchasable and change the description?

File: ModularTegustation/tegu_items/representative/research/kcorp.dm [hunk 2]
```diff
@@ -61,13 +61,28 @@
...
+//ERTs
+/datum/data/lc13research/mobspawner/l1team
+	research_name = "L1 Strike Team"
+	research_desc = "Heads up, Kcorp's given a greenlight on drawing from their more... Physical resources. <br>All they're asking for is a foward investment. You can handle that right?"
+	cost = AVERAGE_RESEARCH_PRICE
+	corp = K_CORP_REP
+	mobspawner_type = /obj/effect/mob_spawn/human/supplypod/r_corp/kcorp_call
+
+/datum/data/lc13research/mobspawner/l3team
+	research_name = "L3 Strike Team"
```
Comment (Kitsunemitsu) [inline ModularTegustation/tegu_items/representative/research/kcorp.dm:83]: These should be Code 3 Strike team

File: ModularTegustation/tegu_items/representative/research/rcorp.dm [hunk 3]
```diff
@@ -1,6 +1,60 @@
...
+	research_desc = "We have a few new weapon designs drawn up using the RED bullets L-Corp is giving us. However we need more funding to kickstart the project, mind handing some over? I'm sure they won't mind."
+	cost = LOW_RESEARCH_PRICE
+	corp = R_CORP_REP
+
+/datum/data/lc13research/redweapon/ResearchEffect(obj/structure/representative_console/caller)
+	ItemUnlock(caller.order_list, "R Corp X-15 Minigun ",	/obj/item/gun/energy/e_gun/rabbit/minigun, 600)
+	ItemUnlock(caller.order_list, "R Corp Lawnmower 2200",	/obj/item/gun/energy/e_gun/rabbitdash/small, 600)
+	ItemUnlock(caller.order_list, "R Corp X-12 Sniper ",	/obj/item/gun/energy/e_gun/rabbitdash/sniper, 600)
+	ItemUnlock(caller.order_list, "R Corp X-13 Beam ",		/obj/item/gun/energy/e_gun/rabbitdash/laser, 600)
+	ItemUnlock(caller.order_list, "R Corp 2300 Shotgun ",	/obj/item/gun/energy/e_gun/rabbitdash/shotgun, 600)
```
Comment (Kitsunemitsu) [inline ModularTegustation/tegu_items/representative/research/rcorp.dm:15]: Same here, the R-Corp Model 2300 works.

File: ModularTegustation/tegu_items/representative/spawners.dm [hunk 4]
```diff
@@ -117,3 +119,26 @@
 	faction = list("neutral", "zwei")
 	icon_state = "zwei"
 	uses = 8	//More because they job
+
+//Kcorp
+/obj/effect/mob_spawn/human/supplypod/r_corp/kcorp_call
+	name = "Kcorp L1 teleport zone"
```
Comment (Kitsunemitsu) [inline ModularTegustation/tegu_items/representative/spawners.dm:125]: Name change here to, to Code 2 and Code 3

File: code/modules/clothing/under/clerksuits.dm [hunk 5]
```diff
@@ -122,7 +122,7 @@
 
 /obj/item/clothing/under/suit/lobotomy/kcorp
 	name = "k-corp officer uniform"
-	desc = "A plain black suit with green tie. Used by K-corp non-combat staff."
```
Comment (Kitsunemitsu) [inline code/modules/clothing/under/clerksuits.dm:125]: This is accurate I think, we have other outfits for K-Corp they're just the old Centcom outfits.

File: ModularTegustation/tegu_items/representative/research/kcorp.dm [hunk 6]
```diff
@@ -61,13 +61,28 @@
...
 
-/datum/data/lc13research/critincrease/ResearchEffect(obj/structure/representative_console/caller)
-	new /obj/item/kcrit(get_turf(caller))
+/datum/data/lc13research/healthboost/ResearchEffect(obj/structure/representative_console/caller)
+	new /obj/item/khpboost(get_turf(caller))
 
+//ERTs
+/datum/data/lc13research/mobspawner/l1team
+	research_name = "L1 Strike Team"
+	research_desc = "Heads up, Kcorp's given a greenlight on drawing from their more... Physical resources. <br>All they're asking for is a foward investment. You can handle that right?"
```
[the remark this answers is not in the document]
Comment (TheBronJameOffical) [reply ModularTegustation/tegu_items/representative/research/kcorp.dm:77]: Apparently they're "Class 1" and "Class 3" staff. I love pm naming schemes.

File: code/modules/clothing/under/clerksuits.dm (hunk 5 above)
Comment (TheBronJameOffical) [reply code/modules/clothing/under/clerksuits.dm:125]: Ah cool, i'll snag them then.

**Review by Kitsunemitsu — APPROVED** [review]
Approved

**Pull request merged**